.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: elixir
 
 elixir:
+  - 1.7
   - 1.6
   - 1.5
-  - 1.4
 
 otp_release:
-  - 20.1
+  - 21.1
+  - 20.3
   - 19.3
   - 18.3
 
 matrix:
   exclude:
     - elixir: 1.6
       otp_release: 18.3
+    - elixir: 1.5
+      otp_release: 21.1
 
 env:
   global:
```

Comment:
I think `sudo: false` can be removed now.  https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration